code/game/turfs/open/floor/prison_floor.dm

```diff
@@ -0,0 +1,81 @@
+#define MAX_PRISON_PLATES 4
+
+/turf/open/floor/prison
+	name = "secure floor"
+	desc = "Prison break-proof!"
+	icon = 'icons/turf/prisonfloor.dmi'
+	icon_state = "prisonfloor4"
+	max_integrity = 500
+	holodeck_compatible = TRUE
+	thermal_conductivity = 0.025
+	heat_capacity = INFINITY
+	floor_tile = /obj/item/stack/sheet/plasteel
+	footstep = FOOTSTEP_PLATING
+	barefootstep = FOOTSTEP_HARD_BAREFOOT
+	clawfootstep = FOOTSTEP_HARD_CLAW
+	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	tiled_dirt = FALSE
+	baseturfs = /turf/open/floor/plating
+	var/plates_type = /obj/item/stack/tile/plasteel
+	var/plates = MAX_PRISON_PLATES
+	var/wrenching = FALSE
+
+/turf/open/floor/prison/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>The reinforcement plates are <b>wrenched</b> firmly in place.</span>"
+
+/turf/open/floor/prison/break_tile()
+	return //unbreakable
+
+/turf/open/floor/prison/burn_tile() // consider changing this
+	return //unburnable
+
+/turf/open/floor/prison/make_plating(force = 0)
+	ScrapeAway(flags = CHANGETURF_INHERIT_AIR)
+	return
```

Comment:
This proc override can be removed as it is the defafult behaviour for make_plating anyway.